server/webdriver/tests/standalone/sdg_sanity.py

```diff
@@ -0,0 +1,407 @@
+      ))
+      return
+
+    # click search to load countries
+    search_container.click()
+
+    # sleep for 500ms for countries to load
+    time.sleep(0.5)
+
+    # countries
+    countries = find_elems(self.driver, By.CSS_SELECTOR,
+                           HomeConstants.COUNTRY_DROPDOWN_CSS_SELECTOR)
+    if len(countries) == 0:
+      self.add_result(fail_result(
+          page,
+          "Countries dropdown not found.",
+      ))
+      return
+
+    # Pass
+    self.add_result(pass_result(page))
+
+  def country(self,
+              country_dcid: str,
+              stat_var: str = None,
+              source: WebPage = None):
+    path = f"countries?p={urllib.parse.quote_plus(country_dcid)}{f'&v={urllib.parse.quote_plus(stat_var)}' if stat_var else ''}"
+    page = self.new_page(PageType.COUNTRY, path, source=source)
+    page.title = f"{country_dcid}{f'({stat_var})' if stat_var else ''}"
+    logging.info("Running: %s", page.url)
+
+    self.driver.get(page.url)
+
+    # sleep for 500ms for page to load
+    time.sleep(0.5)
+
+    # Wait 5 seconds for spinner to disappear
+    try:
+      WebDriverWait(self.driver, 10).until(
+          EC.invisibility_of_element_located(
+              (By.CLASS_NAME, CountryConstants.SPINNER_CSS_CLASS_NAME)))
+    except Exception:
+      self.add_result(fail_result(
+          page,
+          "Timed out.",
+      ))
+
+    goal_cards = find_elems(self.driver, By.CSS_SELECTOR,
+                            CountryConstants.GOAL_CARD_CSS_SELECTOR)
+    if len(goal_cards) == 0:
+      self.add_result(fail_result(
+          page,
+          "No goal cards.",
+      ))
+      return
+
+    # If no target headers => failure
+    target_headers = find_elems(self.driver, By.CLASS_NAME,
+                                CountryConstants.TARGET_HEADER_CSS_CLASS_NAME)
+    if len(target_headers) == 0:
+      self.add_result(fail_result(
+          page,
+          "No target headers.",
+      ))
+      return
+
+    # Pass
+    self.add_result(pass_result(page))
+
+    # if stat_var was already specified, don't navigate further down.
+    if stat_var:
+      return
+
+    stat_vars = []
+
+    def find_stat_vars(parent):
+      expandables = find_elems(parent, By.XPATH,
+                               CountryConstants.EXPANDABLE_GOALS_XPATH)
+      for expandable in expandables:
+        try:
+          # expand
+          expandable.click()
+        except Exception as e:
+          logging.warning("Error expanding a sidebar goal: %s\n%s (%s)\n%s",
+                          expandable, country_dcid, stat_var, e)
+          continue
+
+        # Sleep for 100ms so the elems can expand.
+        time.sleep(0.1)
+        stat_var_elems = find_elems(expandable, By.XPATH,
+                                    CountryConstants.STAT_VAR_XPATH)
+        if stat_var_elems:
+          for stat_var_elem in stat_var_elems:
+            # if the attribute ends with "dc/*", then that is the statvar to navigate to.
```

Comment:
Is the `dc/` true for all SVs?  I thought we may have sdg/ stuff?

Reply:
It seems to be of the form `dc/topic/sdg_*`. e.g. https://datcom-un-sdg.ue.r.appspot.com/UNSDWebsite/undatacommons/sdgs/goals?v=dc%2Ftopic%2Fsdg_6.2.1